src/data/store/WfsFeatures.js

```diff
@@ -218,6 +212,32 @@ Ext.define('GeoExt.data.store.WfsFeatures', {
         return totalCount;
     },
 
+    /**
+     * Sends the sortBy parameter to the WFS Server
+     * If multiple sorters are specified then multiple fields are
+     * sent to the server.
+     * Ascending sorts appends +A and descending sorts +D
+     * E.g. sortBy=attribute1+D,attribute1+A
+     * @private
+     * @return {String} The sortBy string
+     */
+    createSortByParameter: function() {
+
+        var me = this;
+        var sortStrings = [];
+        var direction;
+        var property;
+
+        Ext.each(me.getSorters().items, function(sorter) {
+            // direction will be either A from ASC or D from DESC
+            direction = sorter.getDirection().charAt(0);
+            property = sorter.getProperty();
+            sortStrings.push(Ext.String.format('{0} {1}', property, direction));

```

Comment:
Shouldn't there be a `'+'` between property and direction? Or am I missing something?

Reply:
See comment below